DESCRIPTION

```diff
@@ -2,13 +2,18 @@ Type: Package
 Package: epidatasets
 Title: Epidemiological Data for Delphi Tooling Examples
 Version: 0.0.1
-Authors@R: 
-    person(c("Daniel", "J."), "McDonald", , "daniel@stat.ubc.ca", role = c("cre", "aut"))
+Authors@R: c(
+    person(c("Daniel", "J."), "McDonald", email="daniel@stat.ubc.ca", role = c("aut")),
+    person("Nat", "DeFries", email="ndefries@andrew.cmu.edu", role = c("cre", "aut")),
+    person("Johns Hopkins University Center for Systems Science and Engineering", role = "dtc", comment = "Owner of COVID-19 cases and deaths data sourced from the COVID-19 Data Repository"),
+    person("Johns Hopkins University", role = "cph", comment = "Copyright holder of COVID-19 cases and deaths data sourced from the COVID-19 Data Repository"),
+    person("Carnegie Mellon University Delphi Group", role = "dtc", comment = "Owner of masking and social-distancing data sourced from the COVID-19 Trends and Impacts Survey")
```

Comment:
dtc + cph seems like a nice solution to some difficulties.  We're missing something here for doctor visits data (in `archive_cases_dv_subset_dt`) and `can_prov_cases`; are we able to use this sort of approach for them?

Reply:
Added the attribution for the Canadian data.

Reply:
Waiting on Roni/Carlyn for more info about how to attribute the `doctor-visits` source.

Reply:
From Roni, we are considering Delphi as the data contributor for `doctor-visits`